package.json

```diff
@@ -1,54 +1,64 @@
 {
-  "name": "keep",
+  "name": "keep-website",
+  "description": "The frontend for https://keep.network",
   "version": "0.1.0",
   "private": true,
   "dependencies": {
-    "bootstrap-css-only": "^3.3.7",
-    "classnames": "^2.2.5",
-    "glamor": "^2.20.40",
-    "lodash": "^4.17.13",
-    "moment": "^2.20.1",
-    "pascal-case": "^2.0.1",
-    "prismjs": "^1.7.0",
-    "prop-types": "^15.5.10",
-    "react": "^15.6.1",
-    "react-bootstrap": "^0.31.2",
+    "bootstrap-css-only": "^4.4.1",
+    "bulma": "^0.8.0",
+    "gatsby": "^2.13.31",
+    "gatsby-image": "^2.0.23",
+    "gatsby-plugin-favicon": "^3.1.6",
+    "gatsby-plugin-netlify": "^2.0.6",
+    "gatsby-plugin-netlify-cms": "^4.1.6",
+    "gatsby-plugin-purgecss": "^4.0.0",
+    "gatsby-plugin-react-helmet": "^3.0.4",
+    "gatsby-plugin-sass": "^2.0.7",
+    "gatsby-plugin-sharp": "^2.2.9",
+    "gatsby-remark-copy-linked-files": "^2.0.7",
+    "gatsby-remark-images": "^3.1.6",
+    "gatsby-remark-relative-images": "^0.2.1",
+    "gatsby-source-filesystem": "^2.0.26",
+    "gatsby-transformer-remark": "^2.6.9",
+    "gatsby-transformer-sharp": "^2.1.9",
+    "lodash": "^4.17.15",
+    "lodash-webpack-plugin": "^0.11.4",
+    "netlify-cms-app": "^2.9.6",
+    "netlify-cms-media-library-cloudinary": "^1.3.2",
+    "netlify-cms-media-library-uploadcare": "^0.5.2",
```

Comment:
Do we need these two media library plugins?

Reply:
Mm...I'll look into it! TBH it's copy pasta 🍝 😬

Reply:
I'll take it out.

Reply:
Removed!